networkit/test/test_community.py

```diff
@@ -198,7 +295,18 @@ def testPLP(self):
 			for j in PLPLLP.getMembers(i):
 				reconstructedSet.append(j)
 		self.assertEqual(set(self.LL.iterNodes()), set(reconstructedSet))
+		self.assertIsInstance(PLPL.numberOfIterations(), int)
+		self.assertIsInstance(PLPL.getTiming(), list)
 	
+	def testSpectralPartitioner(self):
+		#small instance
+		specSm = nk.community.SpectralPartitioner(self.L, 3)
+		specSm.run()
+		#medium instance
+		jazz = nk.readGraph("input/jazz.graph",nk.Format.METIS)
+		spec = nk.community.SpectralPartitioner(jazz, 4, balanced=False)
+		spec.run()
```

Comment:
I think we should be more precise when testing this code, since it is mostly Python-code. I would suggest testing for `getPartition()` after `run()`(also inspect the result concerning expectations). If you do this, better als split small and medium instances into two tests.